blockchain/blockchain.go

```diff
@@ -1397,6 +1415,41 @@ func (bc *BlockChain) gcCachedNodeLoop() {
 	}()
 }
 
+func (bc *BlockChain) pruneTrieNodeLoop() {
+	// ReadPruningMarks(1, limit) is very slow because it iterates over the most of MiscDB.
+	// ReadPruningMarks(start, limit) is much faster because it only iterates a small range.
+	startNum := uint64(1)
+
+	bc.wg.Add(1)
```

Comment:
There's no `Wait()` call. What it has an effect without this,?

Reply:
There is https://github.com/klaytn/klaytn/blob/320554640a13fbadf12d42db0a0f91c0edc14386/blockchain/blockchain.go#L1037C14-L1037C14.